### PR TITLE
Update registry.json with KNGHT pool info

### PR DESCRIPTION
**Pull request by knghtpool**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+ "poolId": "41ddaaf2303176365dc36a58483aa8aba694da490d04d84783d418e1",
+ "social": {
+  "facebook": "",
+  "twitter": "CardanoFood",
+  "website": "https://knightsofcardano.com",
+  "reddit": "",  
+  "instagram": " ",
+  "youtube": ""
+ },
+ "telegram": "KnightsOfCardano",
```
Comment (leo42) [inline registry.json:13]: We require a personal Telegram handle(no group or channel) this is to validate voting results.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
 [ 
+ {
+ "ticker": "KNGHT",
+ "poolId": "41ddaaf2303176365dc36a58483aa8aba694da490d04d84783d418e1",
+ "social": {
+  "facebook": "",
+  "twitter": "CardanoFood",
```
[the remark this answers is not in the document]
Comment (knghtpool) [reply registry.json]: Oops! Copy/paste issue, removed that

File: registry.json (hunk 1 above)
Comment (knghtpool) [reply registry.json]: added personal handle

Comment (kiriakos) [reply registry.json]: @knghtpool I see `knghtpool` as the telegram handle. Is that your personal tg account?

Comment (knghtpool) [reply registry.json]: @kiriakos that's right, I did not have a personal telegram handle before setting up the pool so I chose this as my handle. I could change it to something else if preferable

**Review by SinglePoolAlliance — APPROVED** [review]

**Pull request merged**